src/gl_draw.cxx

```diff
@@ -117,30 +117,33 @@ void  gl_font(int fontid, int size) {
 }
 
 #ifndef __APPLE__
-static void get_list(int r) {
-  gl_fontsize->glok[r] = 1;
 #if defined(USE_X11)
 # if USE_XFT
 // FIXME
 # else
+static void get_list(int r) {
+  gl_fontsize->glok[r] = 1;
```

Comment:
This code is completely different in 1.4.0, and I don't think it's worth fixing it in 1.3.x